kitchen.yml

```diff
@@ -58,6 +56,67 @@ suites:
       inspec_tests: [test/integration/openjdk]
       inputs: {java_version: '13'}
 
+  # OpenJDK_PKG
+  - name: openjdk-pkg-7
+    run_list:
+      - recipe[test::openjdk_pkg]
+    attributes: {version: '7'}
+    verifier:
+      inspec_tests: [test/integration/openjdk_pkg]
+      inputs: {java_version: '7'}
+    excludes:
+      - debian-9
+      - debian-10
+
+  - name: openjdk-pkg-8
+    run_list:
+      - recipe[test::openjdk_pkg]
+    attributes: {version: '8'}
+    verifier:
+      inspec_tests: [test/integration/openjdk_pkg]
+      inputs: {java_version: '8'}
+    excludes:
+      - debian-10
+
+  - name: openjdk-pkg-11
+    run_list:
+      - recipe[test::openjdk_pkg]
+    attributes: {version: '11'}
+    verifier:
+      inspec_tests: [test/integration/openjdk_pkg]
+      inputs: {java_version: '11'}
+    excludes:
+      - amazon-linux
+      - debian-9
+
+  - name: openjdk-pkg-13
+    run_list:
+      - recipe[test::openjdk_pkg]
+    attributes: {version: '13'}
+    verifier:
+      inspec_tests: [test/integration/openjdk_pkg]
+      inputs: {java_version: '13'}
+    excludes:
+      - amazon-linux
+      - amazonlinux-2
+      - centos-7
+      - debian-9
+      - debian-10
+
+  - name: openjdk-pkg-latest
+    run_list:
+      - recipe[test::openjdk_pkg]
+    attributes: {version: 'latest'}
+    verifier:
+      inspec_tests: [test/integration/openjdk_pkg]
+      inputs: {java_version: '13'}
+    excludes:
```

Comment:
An includes might just be easier here, and limit it to two platforms. Ubuntu & CentOS?

Reply:
I removed this all together as when I was developing and testing I tested against centos-6 since it was in the vagrant kitchen list. Digging into other OS's seems only 6 lists a latest. Going with ideology of supporting latest and 1 previous OS version I removed centos-6 and added 8. Going to create a new PR for those changes after this one gets merged. That way can use new update and sooner than later and have more discussion on testing and OS support.